docs/src/pages/careers/lead-designer.md

```diff
@@ -1,14 +1,13 @@
 # Lead Designer
 
-<p class="description">You will be accountable for MUI's design execution and manage a small team of designers.</p>
+<p class="description">You will be accountable for MUI's design execution and a small team of designers.</p>
 
 ## Details of the Role
 
-- Location: Remote (preference for UTC-6 to UTC+5).
-- Type of work: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
-- Level: [4 or above](https://docs.google.com/spreadsheets/d/18VvbVCNBRWu-QYPncBl5IWTXTOpD68UQsLsymBvfm88/edit#gid=0).
-- We're a remote company, we prefer asynchronous communication over meetings.
-- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.
+- **Location**: Remote (preference for UTC-6 to UTC+5).
+- **Type of work**: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
+- **Level**: [4 or above](https://docs.google.com/spreadsheets/d/18VvbVCNBRWu-QYPncBl5IWTXTOpD68UQsLsymBvfm88/edit#gid=0).
+- We're a **remote** company, operating mostly asynchronously.
```

Comment:
It sounds fair, could you update the role template https://github.com/mui/material-ui/blob/master/docs/src/pages/careers/ROLE_TEMPLATE.md with the changes that impacts all roles?

Ideally the other open roles too but I could sync if necessary.

Reply:
I'll do the template for now just so this PR doesn't get unnecessarily long!